src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimeZone=UTC</property>

```

Comment:
```suggestion
        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimezone=UTC</property>

```
In your case it works, because your program can work without this parameter